@xen-orchestra/lite/src/components/pool/dashboard/ArrowCollapseButton.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <button class="description-toggle-button">
+    <UiIcon v-if="!isCollapsedRef" :icon="faAngleDown" class="description-toggle-button__icon" />
+    <UiIcon v-else :icon="faAngleUp" class="description-toggle-button__icon" />
+  </button>
+</template>
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+const props = defineProps<{
+  isCollapsed: boolean
+}>()
+
+const isCollapsedRef = computed(() => props.isCollapsed)
+</script>
```

Comment:
As a quick reminder:
- props are directly accessible from template
- we don't use `is` prefix for `boolean` props
- you can use ternary directly in a bind attribute

```suggestion
<template>
  <button class="description-toggle-button">
    <UiIcon :icon="collapsed ? faAngleUp : faAngleDown" class="description-toggle-button__icon" />
  </button>
</template>

<script lang="ts" setup>
import UiIcon from '@core/components/icon/UiIcon.vue'
import { faAngleDown, faAngleUp } from '@fortawesome/free-solid-svg-icons'

defineProps<{
  collapsed: boolean
}>()
</script>
```